integrations/lib/testing/integration/bootstrap.go

```diff
@@ -45,7 +45,8 @@ func (bootstrap *Bootstrap) AddUserWithRoles(name string, roles ...string) (type
 }
 
 func (bootstrap *Bootstrap) AddRole(name string, spec types.RoleSpecV6) (types.Role, error) {
-	role, err := types.NewRole(name, spec)
+	// TODO: Remove this once Test Integration is updated to build tctl.
```

Comment:
Can you create an issue for that, assign it to the owner, and link it here? Otherwise, this will never be addressed.

Reply:
done in [0911cb0](https://github.com/gravitational/teleport/pull/26583/commits/0911cb035617ca17e4f96e0cfecbcb3aac82c8f0)